### PR TITLE
Plot abundance

### PR DESCRIPTION
**Pull request by YagmurSimsekk**
Now the list is turned into single plot.

### CODE REVIEW

File: R/plotAbundance.R [hunk 1]
```diff
@@ -129,6 +133,7 @@ setGeneric("plotAbundance", signature = c("x"),
 #' @rdname plotAbundance
 #' @importFrom scater plotExpression
 #' @importFrom ggplot2 facet_wrap
+#' @importFrom gridExtra grid.arrange
```
Comment (antagomir) [inline R/plotAbundance.R:136]: We recommend the **patchwork** pkg for multi-panel plots. Could you change this?

**Review by antagomir — COMMENTED** [review]
Thanks. And this is for issue #44

File: R/plotAbundance.R [hunk 2]
```diff
@@ -18,92 +18,96 @@
...
-#'   
-#' @param order_rank_by How to order abundance value: By name (\dQuote{name}) 
+#'
+#' @param order_rank_by How to order abundance value: By name (\dQuote{name})
 #' for sorting the taxonomic labels alphabetically, by abundance (\dQuote{abund}) to
 #' sort by abundance values or by a reverse order of abundance values (\dQuote{revabund}).
-#'  
-#'   
+#'
+#'
```
Comment (TuomasBorman) [inline R/plotAbundance.R:32]: Additonal lines could be deleted, here is two lines and others have one between @params

**Review by TuomasBorman — COMMENTED** [review]
Hi,

looks good and solid. 

However, I am still thinking if we should only update manuals

1. This creates a new dependency
2. For example, if user wants to change colors of colData-plot (in the example case: if user wants to specify colors of sample types "Ocean", "Tongue", and so on), is it too complictaed when returned plot is patchwork, not ggplot?
3.  patchwork(plot) is relatively simple task
4. User might prefer some other tools like gridExtra

-Tuomas

**Comment by antagomir** [comment]
I agree with @TuomasBorman but now investigated this in more detail.

The very first implementation by @FelixErnst (see [commit history](https://github.com/microbiome/miaViz/commits/master/R/plotAbundance.R) seems to include a case for combining plots, with `@importFrom cowplot plot_grid`. Hence, it is intended that plots are combined internally. We can keep the original version and dependency on that, if there is no reason to change. 

The problem was reported in issue #44 and the problem is that the function does not work as expected in a certain case as it returns two plots that are plotted separately, and not a single combined plot for the sample labels are shown below the main figure (in contrast to what the documentation says). 

Can we update this PR so that it solves the problem that is described in issue #44 so that the function will return, for that example, the intended type of plot as explained on the manpage? If this appears to be challenging, we can discuss more online.

**Comment by YagmurSimsekk** [comment]
It has been updated to match what is explained in the man page. PR can be merged.

**Comment by FelixErnst** [comment]
Doesn't solve the issue and forces combination of plot reducing massively the flexibility of the function

**Comment by antagomir** [comment]
Yes. The best solution could be to open an entirely new PR that addresses the issue described in #44

**Comment by FelixErnst** [comment]
The issues in #44 are addressed and solved. Nothing to do anymore or did I overlook something?

**Pull request closed without merging**